app/views/contribute.html

```diff
@@ -0,0 +1,186 @@
+{% extends 'layout.html' %}
+{% block pageTitle %}
+  Contribute - NHS prototype kit
+{% endblock %}
+{% block beforeContent %}
+  {{ breadcrumb({ href: "/", text: "Home" }) }}
+{% endblock %}
+{% block content %}
+
+{# the board is a long string to this is made a variable for consistency #}
+{% set board = "https://github.com/orgs/nhsuk/projects/24/views/1" %}
+{# the labels string is also tricky to making a variable #}
+{% set label = "?filterQuery=label%3A" %}
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-two-thirds">
+    <h1 id="contribute-to-the-nhs-prototype-kit">
+      Contribute
+    </h1>
+    <p>
+      The NHS prototype kit is a community project supported by the NHS service
+      manual team. Anyone can help make it better.
+    </p>
+    <p>
+      <a href="{{board}}">View project board on GitHub</a>
+    </p>
+    <h2 id="why-contribute-to-the-kit">Why contribute to the kit</h2>
+    <p>If you contribute to the kit:</p>
+    <ul>
+      <li>
+        this keeps it up-to-date as an accessible and open way to make HTML
+        prototypes of NHS services
+      </li>
+      <li>
+        you will become part of an active and supportive community that will
+        help you learn and develop your skills with the kit
+      </li>
+      <li>
+        we will credit you in our update notes and other supporting information
+        unless you tell us not to (for example, because you want to stay
+        anonymous)
+      </li>
+    </ul>
```

Comment:
Reviewing the page again, there are 6 separate bullet lists, and I wonder whether it’d be better to reduce them and only use bullet lists where each item is shorter and more comparable?  This one could be paragraphs instead?

```suggestion
    <p>Help keep the kit up-to-date as an accessible and open way to make HTML prototypes of NHS services.</p>
    <p>By contributing you will become part of an active and supportive community that will help you learn and develop your skills with the kit.</p>
    <p>We will credit you in our update notes and other supporting information, unless you tell us not to.</p>
```